automatic/brave/brave.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>brave</id>
+    <version>0.0</version>
+    <title></title>
+    <description></description>
+    <iconUrl></iconUrl>
```

Comment:
Since you're setting the icon url in the update script, we'll need to exclude the check of this package for the automatically updating of icon urls.
Add a comment with the following text `<!-- IconUrl: Skip check -->`